src/provisioner/provisioner.py

```diff
@@ -56,27 +59,57 @@ def config_apply(solution_conf_url: str, cortx_conf_url: str = None):
 
     @staticmethod
     def config_apply_cortx(cortx_config_store):
-        # TODO: To convert CORTX config
-        cortx_config_store.set('cortx', Conf.get(CortxProvisioner._solution_index, 'cortx'))
+        """ Convert CORTX config into confstore keys """
+        cortx_config = Conf.get(CortxProvisioner._solution_index, 'cortx')
+        cs = CortxConfig(cortx_config)
+        cs.save(cortx_config_store)
 
     @staticmethod
     def config_apply_cluster(cortx_config_store):
         node_map = {}
-        node_types = Conf.get(CortxProvisioner._solution_index, 'cluster>node_types')
-        for node_type in node_types:
-            node_map[node_type['name']] = node_type
-
-        storage_sets = Conf.get(CortxProvisioner._solution_index, 'cluster>storage_sets')
-        nodes = []
-        for storage_set in storage_sets:
-            for node in storage_set['nodes']:
-                node_type = node['type']
-                node = dict(node_map[node_type], **node)
-                node['storage_set'] = storage_set['name']
-                nodes.append(node)
-
-        cs = CortxCluster(nodes)
-        cs.save(cortx_config_store)
+        try:
+            node_types = Conf.get(CortxProvisioner._solution_index, 'cluster>node_types')
+            for node_type in node_types:
+                node_map[node_type['name']] = node_type
+
+            cluster_id = Conf.get(CortxProvisioner._solution_index, "cluster>id")
+            if cluster_id is None:
+                raise CortxProvisionerError(
+                    errno.EINVAL,
+                    f'cluster_id property is unspecified for cluster.')
+
+            cluster_name = Conf.get(CortxProvisioner._solution_index, "cluster>name")
```

Comment:
Please use ' ' consistently instead of " ". Avoid mix.

Reply:
Done.